### ISSUE TITLE
Anime-Planet importer maintainer needed

### ISSUE THREAD

**Issue by hohMiyazawa** [hohMiyazawa/Automail#41]
The A-P importer worked when I wrote it.

Does it work now? Probably. I have no idea.

I would like to have someone that could infrequently check if it still works. You will need:
 - To be familiar with Anime-Planet (I'm not)
 - Have some exported list data from Anime-Planet
 - Have a dummy Anilist account for testing.

**Comment by SSgumS** [comment]
I tested it today with both v9.94.2 and v9.96.4.
Unfortunately it doesn't work. The exported list is for today.

[Exported Anime List](https://files.catbox.moe/ppo4fn.json)
[Errors appeared on console when importing to AniList](https://files.catbox.moe/gq041q.png)

**Comment by hohMiyazawa** [comment]
Thanks for the heads-up!

**Comment by hohMiyazawa** [comment]
Turns out there was some missing error handling for when the search engine doesn't return any hits at all.
Should be running in v9.96.5

**Comment by SSgumS** [comment]
Thanks a lot. it's now working. but I think still there is some small issues.
For example my list has 337 animes. but the script detects just 295 animes. To paraphrase, Haikyuu!! Second Season has not been detected and it's not imported.

If you want to see my imported list, you can use this exported json file from AniList: https://files.catbox.moe/q2ns0o.json

**Comment by hohMiyazawa** [comment]
Yup, title matching is an imprecise science. (hence why this thread exists)
Will try to find out what titles are missing.

**Comment by SSgumS** [comment]
That's a relief. Thanks.
I think for now, it's best to list the names that couldn't be matched to anything. That way user can manually add them later. I saw some matchings by correctness probability showed by colors but unfortunately there was no report on the names that totally didn't get matched.
